cso.dat

```diff
@@ -0,0 +1,8 @@
+V1
+2:PatchEntityCtor,v
+2:context,u
+2:attrib,Q
+2:f_EntityData,B
+2:f_Level_temporaryEntityData,M
+2:cursor,G
+2:instr,g
```

Comment:
?

Reply:
This was done by Visual Studio and honestly I have no fucking clue.